app/controllers/food_inventories_controller.rb

```diff
@@ -0,0 +1,30 @@
+class FoodInventoriesController < ApplicationController
+  def new
+    @food_inventory = FoodInventory.new
+    @inventory_id = params[:inventory_id]
+  end
+
+  def create
+    @food_inventory = FoodInventory.new(food_inventory_params)
+    if @food_inventory.save
+      flash[:success] = 'Food was successfully added to the inventory.'
+      redirect_to inventory_path(id: @food_inventory.inventory_id)
+    else
+      render :new, alert: 'Failed to add ingredient'
+    end
+  end
+
+  def destroy
+    @food_inventory = FoodInventory.find(params[:id])
+
+    @food_inventory.destroy!
+
+    redirect_to inventory_path(@food_inventory.inventory_id)
+  end
```

Comment:
- [ ] @Team it's always a good practice, to have success and failure paths for creating, updating, and destroying actions, whenever an action fails and our app doesn't have the option to refer the user it might seem like the app is broken.